console/console-init/ui/src/modules/device/components/DeviceListAlert/DeviceListAlert.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { DeviceListAlert, IDeviceListAlertProps } from "./DeviceListAlert";
+
+describe("Alert for DeviceListPage", () => {
```

Comment:
follow some standard to write unit test.
1. Add component at top level describe 
describe("<DeviceListAlert/>", () => {}
2. use It instead test as highly used in react app.
3. Start your test information in It/test with should 
it("should render an alert on top of  DeviceListPage", () => {}

Nothing is logical wrong in your test. just lets try to follow some standard.

Reply:
Currently the ui is using `it` and `test` interchangeably. We should address that in near future I think and stick to only `it`.